### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by KovalevArtem** [filebot/website#2]
I am saddened that the FileBot is in English only.
I suggest to localize the program and site.
I am ready to assist in the translation into Russian.
(For example, you can use the service Crowdin, Transifex or any other)

**Comment by rednoah** [comment]
Sorry, supporting I18N in FileBot is not planned:

* Many of the databases only support English, or have most of their information available in English only
* Most of the content being renamed is going to be in English
* Plex requires an English file and folder name structure
* Keeping all translations up to date as new features are added would be difficult
* I can't personally verify the quality of a given translation
* The level of English required to operate FileBot is very basic
* Customer support can only be supported in languages that I speak, so best to keep it all English for the benefit of international users

The situation might change over time, and perhaps in the future I18N will make sense.

TL;DR FileBot users typically speak English well.
